internal/cmd/cmd_test.go

```diff
@@ -354,6 +580,20 @@ func assertGRPC(t *testing.T, expectedExitCode int, expectedLinePrefixes string,
 	assertDoStdin(t, strings.NewReader(jsonData), expectedExitCode, expectedLinePrefixes, "grpc", filePath, excitedTestCase.Address(), method, "-")
 }
 
+func assertRegexp(t *testing.T, expectedExitCode int, expectedRegexp string, args ...string) {
+	stdout, exitCode := testDo(t, args...)
+	assert.Equal(t, expectedExitCode, exitCode)
+	matched, err := regexp.MatchString(expectedRegexp, stdout)
+	assert.NoError(t, err)
+	assert.True(t, matched, stdout)
```

Comment:
Do you think it's overkill to include the `expectedRegexp` in the `assert.True` failure message?

Reply:
Yea I can do that, give me a few mins

Reply:
np - this might also apply to the other test helpers.

Reply:
The other helper I see right now prints the output as the message, which is basically what I want, is that OK?